apps/services/handlers.py

```diff
@@ -277,3 +278,38 @@ async def read_request_details(university_id: int, user_request_id: int, user=De
         "data": response,
         "message": "Got request details"
     }
+
+
+@services_router.post("/{user_request_id}/student-accommodation/{university_id}/",
+                      name="count_student_accommodation",
+                      response_model=JSENDOutSchema[CountHostelAccommodationIn],
+                      tags=["Student dashboard"])
+async def count_student_accommodation(university_id: int, user_request_id: int,  hostel: CountHostelAccommodationIn,
+                                      user=Depends(get_current_user)):
+    """
+    **Count Student Accommodation**
+
+    **Path**:
+    - **university_id**: user university id
+    - **user_request_id**: user request id
+
+    **Input**:
+    - **start_date_accommodation**: start date of user accommodation
+    - **end_date_accommodation**: end(last) date of user accommodation
+    - **bed_place_id**: user bed place id
+
+    **Return**: summa of accommodation, by month price, dates differance and bed place type
+    """
+    hostel_query = select(hostel_accommodation_view).where(hostel_accommodation_view.c.user_request_id == user_request_id,
+                                                           hostel_accommodation_view.c.university_id == university_id,)
+
+    bed_place_query = select(BedPlace).where(BedPlace.bed_place_id == hostel.bed_place_id)
+
+    hostel_info = await database.fetch_one(hostel_query)
+    bed_place_info = await database.fetch_one(bed_place_query)
+
+    return {
+        "data": hostel_info,
+        "message": f"Result summa of student accommodation: "
```

Comment:
Message shouldn't contain data, only message that "Cost of hostel accommodation of student was counted successfully"